app/src/main/java/com/easyfitness/DAO/bodymeasures/BodyPartExtensions.java

```diff
@@ -135,6 +138,8 @@ public static int getBodyLogoID(int pBodyID) {
                 return R.drawable.ic_hip_measure;
             case OBLIQUES:
                 return R.string.obliques;
+            case GLUTES:
```

Comment:
Not required for muscles.

Reply:
Am I right in saying that `getBodyStringID()` is only ever called with muscles that can be measured, and with the units such as `WEIGHT`, `FAT`, `BONES` etc?

If my understanding is correct, since `GLUTES` aren't in the list of measurable muscles, should I remove the addition of `GLUTES` to `getBodyStringID()` as well?

Reply:
Thinking about it, you are right. Having the GLUTES in the getBodyStringID is useless but has no impact.

Reply:
In that case, if you're happy with the rest of this PR, once this is merged I can raise another one to refactor this section a little bit.